contracts/src/staking/DepositContract.sol

```diff
@@ -94,6 +113,31 @@ abstract contract DepositContract is IDepositContract, ERC165 {
         }
     }
 
+    function updateOperator(
```

Comment:
@neverDefined are we okay with one step operator change process?

Reply:
two step would be good, on updates only though